### PR TITLE
Enable test_assert[_nested]

### PR DESCRIPTION
**Pull request by quintinwang5**

### CODE REVIEW

**Review by pbchekin — APPROVED** [review]

**Review by etiotto — CHANGES_REQUESTED** [review]
Just one concern. We need to be able to run all tests using also the script "test-triton.sh" on all build machines and also on our development machines. We shouldn't merge this PR untill that is possible.

**Comment by pbchekin** [comment]
> Please also remove `pytest.skip` for `test_scan2d_broadcast` in `intel-xpu-backend-for-triton/python/test/unit/operators/test_inductor.py`.

Does #385 do this?

File: .github/dockerfiles/runner-base/Dockerfile [hunk 1]
```diff
@@ -18,7 +18,6 @@ RUN set -ex; \
       level-zero \
       level-zero-dev libigc-dev intel-igc-cm libigdfcl-dev libigfxcmrt-dev \
     ; \
-    apt install -y --no-install-recommends --allow-downgrades --fix-missing libigc1=1.0.14828.26-736~22.04; \
```
Comment (etiotto) [inline .github/dockerfiles/runner-base/Dockerfile:21]: The changes to the docker image are being don in a different PR.

Comment (pbchekin) [reply .github/dockerfiles/runner-base/Dockerfile:21]: Yes, we can do this in any order depending on what PR is merged first.

**Review by whitneywhtsang — APPROVED** [review]

**Review by etiotto — APPROVED** [review]

**Pull request merged**